homeassistant/components/analytics/analytics.py

```diff
@@ -0,0 +1,209 @@
+        """Load preferences."""
+        self._data = await self._store.async_load() or {}
+        if self.supervisor:
+            supervisor_info = hassio.get_supervisor_info(self.hass)
+            if not self.onboarded:
+                # User have not configured analytics, get this setting from the supervisor
+                if (
+                    supervisor_info[ATTR_DIAGNOSTICS]
+                    and AnalyticsPreference.DIAGNOSTICS not in self.preferences
+                ):
+                    self._data.setdefault(ATTR_PREFERENCES, []).append(ATTR_DIAGNOSTICS)
+                elif (
+                    not supervisor_info[ATTR_DIAGNOSTICS]
+                    and AnalyticsPreference.DIAGNOSTICS in self.preferences
+                ):
+                    self._data.setdefault(ATTR_PREFERENCES, []).remove(ATTR_DIAGNOSTICS)
+
+    async def save_preferences(self, preferences: List[AnalyticsPreference]) -> None:
+        """Save preferences."""
+        self._data[ATTR_PREFERENCES] = preferences
+        self._data[ATTR_ONBOARDED] = True
+        await self._store.async_save(self._data)
+
+        if self.supervisor:
+            await hassio.async_update_diagnostics(
+                self.hass, AnalyticsPreference.DIAGNOSTICS in self.preferences
+            )
+
+    async def send_analytics(self, _=None) -> None:
+        """Send analytics."""
+        supervisor_info = None
+
+        if not self.onboarded or AnalyticsPreference.BASE not in self.preferences:
+            LOGGER.debug("Nothing to submit")
+            return
+
+        huuid = await self.hass.helpers.instance_id.async_get()
+
+        if self.supervisor:
+            supervisor_info = hassio.get_supervisor_info(self.hass)
+
+        system_info = await async_get_system_info(self.hass)
+        integrations = []
+        addons = []
+        payload: dict = {
+            ATTR_HUUID: huuid,
+            ATTR_VERSION: HA_VERSION,
+            ATTR_INSTALLATION_TYPE: system_info[ATTR_INSTALLATION_TYPE],
+        }
+
+        if supervisor_info is not None:
+            payload[ATTR_SUPERVISOR] = {
+                ATTR_HEALTHY: supervisor_info[ATTR_HEALTHY],
+                ATTR_SUPPORTED: supervisor_info[ATTR_SUPPORTED],
+            }
+
+        if (
+            AnalyticsPreference.USAGE in self.preferences
+            or AnalyticsPreference.STATISTICS in self.preferences
+        ):
+            configured_integrations = await asyncio.gather(
+                *[
+                    async_get_integration(self.hass, domain)
+                    for domain in self.hass.config.components
+                    # Filter out platforms.
+                    if "." not in domain
+                ]
+            )
+
+            for integration in configured_integrations:
+                if (
+                    integration.disabled
+                    or integration.domain in INGORED_DOMAINS
+                    or not integration.is_built_in
```

Comment:
I would still like to know how many people run `default_config`,

Reply:
You will, the informed are being removed